src/Reveal.Sdk.Dom.Tests/Visualizations/VisualizationFixture.cs

```diff
@@ -0,0 +1,445 @@
+        {
+            // Arrange
+            var dataSourceItem = new DataSourceItem();
+            var visualization = new Mock<Visualization>("testTitle", dataSourceItem) { CallBase = true }.Object;
+
+
+            // Act
+            visualization.IsTitleVisible = true;
+
+            // Assert
+            Assert.True(visualization.IsTitleVisible);
+
+            // Act
+            visualization.IsTitleVisible = false;
+
+            // Assert
+            Assert.False(visualization.IsTitleVisible);
+        }
+
+        [Fact]
+        public void ColumnSpan_GetSetCorrectly_WhenUsed()
+        {
+            // Arrange
+            var dataSourceItem = new DataSourceItem();
+            var visualization = new Mock<Visualization>("testTitle", dataSourceItem) { CallBase = true }.Object;
+
+
+            // Act
+            visualization.ColumnSpan = 10;
+
+            // Assert
+            Assert.Equal(10, visualization.ColumnSpan);
+        }
+
+        [Fact]
+        public void RowSpan_GetSetCorrectly_WhenUsed()
+        {
+            // Arrange
+            var dataSourceItem = new DataSourceItem();
+            var visualization = new Mock<Visualization>("testTitle", dataSourceItem) { CallBase = true }.Object;
+
+
+            // Act
+            visualization.RowSpan = 10;
+
+            // Assert
+            Assert.Equal(10, visualization.RowSpan);
+        }
+
+        [Fact]
+        public void Description_GetSetCorrectly_WhenUsed()
+        {
+            // Arrange
+            var dataSourceItem = new DataSourceItem();
+            var visualization = new Mock<Visualization>("testTitle", dataSourceItem) { CallBase = true }.Object;
+            var testDescription = "testDescription";
+
+            // Act
+            visualization.Description = testDescription;
+
+            // Assert
+            Assert.Equal(testDescription, visualization.Description);
+        }
+
+        [Fact]
+        public void FilterBindings_EqualDataDefinitionBindings_WhenUsed()
+        {
+            // Arrange
+            var dataSource = new DataSource();
+            var dataSourceItem = new DataSourceItem
+            {
+                DataSource = dataSource,
+                HasTabularData = true,
+                Fields = new List<IField>
+                {
+                    new TextField("A"),
+                    new TextField("B"),
+                    new TextField("C"),
+                    new TextField("D"),
+                }
+            };
+            var mock = new Mock<Visualization>("testTitle", dataSourceItem) { CallBase = true };
+            var visualization = mock.Object;
+
+            // Act
+            var filterBindings = visualization.FilterBindings;
+
+            // Assert
+            Assert.Equal(((DataDefinitionBase)visualization.DataDefinition).Bindings.Bindings, filterBindings);
+        }
+
+        [Fact]
+        public void GetDataSourceItem_ReturnsCorrectDataSourceItem_WhenUseTabularData()
```

Comment:
We need another unit test / or Theory to test the case that the DataDefinition is not TabularDataDefinition